src/helpers/execute.ts

```diff
@@ -67,43 +65,28 @@ export async function execute<SuccessResponseT>(
         : undefined),
       ...shopIdHeader,
       ...additionalHeaders,
+      ...(auth && auth.type === 'token' ? {'X-Access-Token': auth.token} : {}),
     },
-    url,
     method: bapiCall.method,
-    data:
+    body:
       bapiCall.method === 'POST' || bapiCall.method === 'PATCH'
         ? bapiCall.data
         : undefined,
-    validateStatus: acceptAllResponseCodes
```

Comment:
The validate status logic is missing right?

Reply:
Oops. Added that back